shared/video/_call-type-settings.md

```diff
@@ -0,0 +1,111 @@
+| `quality`    | String `audio-only`, `360p`, `480p`, `720p`, `1080p`, `1440p`                                                           | Defines the resolution of the recording                                                                                                                        |
+| `audio_only` | `boolean`                                                                                                               | If `true` the recordings will only contain audio                                                                                                               |
+| `layout`     | object, for more information see the [API docs](https://getstream.io/video/docs/api/recording/calls/#recording-layouts) | Confoguration options for the recording application                                                                                                            |
+
+### Broadcasting
+
+| Setting Name | Type                  | Description                             |
+| ------------ | --------------------- | --------------------------------------- |
+| `enabled`    | Boolean               | Defines whether broadcasting is enabled |
+| `hls`        | HLS Settings (object) | Settings for HLS broadcasting           |
+
+### HLS Settings
+
+| Setting Name     | Type                                                          | Description                                                       |
+| ---------------- | ------------------------------------------------------------- | ----------------------------------------------------------------- |
+| `enabled`        | Boolean                                                       | Defines whether HLS is enabled or not                             |
+| `auto_on`        | Boolean                                                       | When true HLS streaming will start as soon as users join the call |
+| `quality_tracks` | String `audio-only`, `360p`, `480p`, `720p`, `1080p`, `1440p` | The tracks to publish for the HLS stream (up to three tracks)     |
+
+### Geofencing
+
+| Setting Name | Type                                                                                                                                                                                      | Description                                                     |
+| ------------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------- |
+| `names`      | List of one or more of these strings `european_union`, `iran_north_korea_syria_exclusion`, `china_exclusion`, `russia_exclusion`, `belarus_exclusion`, `india`, `united_states`, `canada` | The list of geofences that are used for the calls of these type |
```

Comment:
Should we describe here in more detail what it means?

E.g. `european_union` will mean that it's only inside the EU or that the EU is blocked?
Same with the `*_exclusion` types. Could lead to confusion, right?

Reply:
Link added in this commit: https://github.com/GetStream/stream-chat-docusaurus-cli/pull/115/commits/da45fa08dd82add4b4b4bce034fd49bf14a714a2 (the commit is noisy with some reformats)